builtin/logical/pki/acme_wrappers.go

```diff
@@ -390,7 +390,7 @@ func isAcmeDisabled(sc *storageContext, config *acmeConfigEntry, policy EabPolic
 		return true
 	}
 
-	if disableAcmeRaw := os.Getenv("VAULT_DISABLE_PUBLIC_ACME"); disableAcmeRaw != "" {
```

Comment:
Should we call `isPublicACMEDisabledByEnv`? If we update that function to error, and log the error in this variant and error in the config case, that'd be nicest on the user that their envvar is bad.

Reply:
Not 100% what you said but I've updated to combine the logic within https://github.com/hashicorp/vault/pull/20600/commits/91f8a9d6f7497d79a86cc78b9b2711d43eab4a6a Let me know if I've badly interpreted your comment.